### PR TITLE
#70 enable SSO + #189 contain f8

### PR DESCRIPTION
**Pull request by anshprat**
#70  Enables SSO login for workplace via jumpcloud. More domains can be added 
to SSO_DOMAINS as required.

Also addes f8.com for #189

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -3,21 +3,24 @@ const FACEBOOK_CONTAINER_NAME = "Facebook";
 const FACEBOOK_CONTAINER_COLOR = "blue";
 const FACEBOOK_CONTAINER_ICON = "briefcase";
 const FACEBOOK_DOMAINS = [
-  "facebook.com", "www.facebook.com", "facebook.net", "fb.com", 
+  "facebook.com", "www.facebook.com", "facebook.net", "fb.com",
```
Comment (nemoload) [inline src/background.js]: The spaces were left intentionally, you shouldn't remove them.

Comment (nemoload) [reply src/background.js]: Also merge the two commits into one commit for better history management.

Comment (anshprat) [reply src/background.js]: spaces removal was by IDE, will fix. merging the commits.

Comment (anshprat) [reply src/background.js]: @nemoload - can you tell me why do we have spaces in the end only in some lines and not all of the lines ( example - line 7,8,11,15,17  don't have spaces in end?

**Comment by groovecoder** [comment]
Thank you for the contribution.

We are planning to implement a solution for SSO, and we may use a white-listing approach like this. But, we are still defining our UX + UI for it.

That being the case, I'm going to close this for now, but we may be able to use this code in future changes addressing SSO.

**Pull request closed without merging**